src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -128,278 +243,205 @@ function mapStateToProps() {
 /**
  * The Loadout Optimizer screen
  */
-class LoadoutBuilder extends React.Component<Props, State> {
-  private subscriptions = new Subscriptions();
-  private filterItemsMemoized = memoizeOne(filterItems);
-  private filterSetsMemoized = memoizeOne(filterGeneratedSets);
-  private processMemoized = memoizeOne(process);
-  private getEnabledStats = memoizeOne(
-    (statFilters: Readonly<{ [statType in StatTypes]: MinMaxIgnored }>) =>
-      new Set(statKeys.filter((statType) => !statFilters[statType].ignored))
+function LoadoutBuilder({
+  account,
+  storesLoaded,
+  stores,
+  isPhonePortrait,
+  items,
+  defs,
+  searchConfig,
+  filters,
+  location,
+}: Props) {
+  // Memoizing to ensure these functions are only created once
+  const [filterItemsMemoized, filterSetsMemoized, processMemoized, getEnabledStats] = useMemo(
+    () => [
+      memoizeOne(filterItems),
```

Comment:
This is cool, but I bet you could actually just use `useMemo` at the invocation site instead of memoizing creating a memoized version of the function.

Reply:
The issue is the try/catch block they are used in, eslint complains as they are used conditionally. That won't be needed anymore once I move process to a hook, so this was an intermediate step. Once the web worker is done I will be able to remove the try/catch altogether from my understanding of how this works.

Let me know if there is another approach you want to take and I can do it in this PR.